alkira/resource_alkira_connector_aruba_edge.go

```diff
@@ -174,7 +174,28 @@ func resourceConnectorArubaEdgeRead(d *schema.ResourceData, m interface{}) error
 		return err
 	}
 
-	setArubaEdgeResourceFields(connector, d)
+	arubaEdgeMappings, err := deflateArubaEdgeVrfMapping(connector.ArubaEdgeVrfMapping, m)
+	if err != nil {
+		return err
```

Comment:
Do we really want to just quit here if we failed to get VRF mapping.. or we want to show as much as possible and display errors that mapping was screwed up... I was thinking about this case.. but didn't get definitive answer...

Reply:
I would say yes. The deciding factor is likely how important that field is to the overall functionality. This is a required field (if memory serves the docs are missing required tags) so I would say yes fail and display the error.